ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -75,6 +75,25 @@
                         {"$ref": "#/$defs/none-action"},
                         {"$ref": "#/$defs/shutdown-action"}
                     ]
+                },
+		"once_within" : {
+			"type": "string",
+			"minLength": 6,
+			"pattern": "^\\d+\\s{1}(milliseconds|seconds|minutes|hours|days)$"
+		},
+		"time_window" : {
+			"type": "string",
+			"minLength": 6,
+			"pattern": "^\\d+\\s{1}(milliseconds|seconds|minutes|hours|days)$"
+		},
+		"timed_out" : {
+			"type": "string",
+			"minLength": 6,
+			"pattern": "^\\d+\\s{1}(milliseconds|seconds|minutes|hours|days)$"
+		},
+                "unique_attributes": {
+                    "type": "array",
+                    "items": { "type": "string" }
```

Comment:
It should be in `once_within` isn't?

Reply:
@Alex-Izquierdo unique_attributes is an optional attribute used in conjunction with once_within. Not sure how I would nest it within once_within.

Reply:
I think you need this: https://json-schema.org/understanding-json-schema/reference/conditionals.html